aiogram/utils/enum.py

```diff
@@ -0,0 +1,7 @@
+import enum
+
+
+class AutoName(enum.Enum):
```

Comment:
I think it would be better to make this inherit from `str, enum.Enum`, otherwise `self.lower()` may not work if one forgets to add `str` as a subclass before `AutoName`.

```suggestion
class AutoName(str, enum.Enum):
```